Lib/ipaddress.py

```diff
@@ -1519,7 +1519,13 @@ def __init__(self, address, strict=True):
         # Constructing from a tuple (addr, [mask])
         elif isinstance(address, tuple):
             addr = address[0]
-            mask = address[1] if len(address) > 1 else self._max_prefixlen
+            if len(address) > 1:
+                if isinstance(address[1], int) and 0 < address[1] > 32:
```

Comment:
```suggestion
                if isinstance(address[1], int) and address[1] > 32:
```
if `address` is larger than 32, it is larger than 0.